src/connectors/stats/__tests__/connectStats-test.ts

```diff
@@ -65,13 +66,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/stats/js/#c
       const renderFn = jest.fn();
       const unmountFn = jest.fn();
       const createStats = connectStats(renderFn, unmountFn);
-      const stats = createStats();
+      const stats = createStats({});
```

Comment:
we should still have a test somewhere instantiating createStats() without argument, is there one?

Reply:
done in 349dcc22